src/borg/_hashindex.c

```diff
@@ -369,45 +417,90 @@ hashindex_write(HashIndex *index, const char *path)
 static const void *
 hashindex_get(HashIndex *index, const void *key)
 {
-    int idx = hashindex_lookup(index, key);
+    int idx = hashindex_lookup(index, key, NULL);
     if(idx < 0) {
         return NULL;
     }
     return BUCKET_ADDR(index, idx) + index->key_size;
 }
 
+inline void
+memswap(void *a, void *b, void *tmp, size_t entry_size) {
+    memcpy(tmp, a, entry_size);
+    memcpy(a, b, entry_size);
+    memcpy(b, tmp, entry_size);
```

Comment:
it's not a problem as long as we build on wheezy (or on anything else with < 2.14).